docs/oak_functions_abi.md

```diff
@@ -97,6 +97,25 @@ client.
 - `param[0]: buf_ptr: i32`: Address of the response buffer.
 - `param[1]: buf_len: i32`: Number of bytes of the response buffer.
 
+- `result[0]: i32`: Status of operation as
+  [`OakStatus`](https://github.com/project-oak/oak/blob/main/oak_functions/proto/abi.proto)
+
+### `write_log_message`
+
+Writes a log message. This message will be considered sensitive, so will only be
+logged if the `oak_unsafe` feature is enabled.
+
+The low-level operation involves reading the response from the WebAssembly
+module's memory. The system will attempt to interpret the bytes as a UTF-8
+encoded string. If the decoding is successful, the string will be logged as a
+debug message. If the bytes are not a valid UTF-8 string a warning message
+containing the UTF-8 decoding error and the raw bytes will be logged.
```

Comment:
(nit) perhaps stick to present tense throughout instead of switching between present and future.

Reply:
Done.